### PR TITLE
Create motion.create action and add to calculate state_id

### PR DESCRIPTION
**Pull request by reiterl**
Add draft motion.create action and tests.
Update motion.create: add state_id calculation.
Add two tests.

### CODE REVIEW

File: openslides_backend/action/motion/create.py [hunk 1]
```diff
@@ -1,7 +1,50 @@
...
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        # calculate state_id from workflow_id
+        workflow_id = instance.pop("workflow_id", None)
+        if workflow_id is None:
+            meeting = self.database.get(
+                FullQualifiedId(Collection("meeting"), instance["meeting_id"]),
+                ["motions_default_workflow_id"],
+            )
+            workflow_id = meeting.get("motions_default_workflow_id")
+        if workflow_id:
```
Comment (FinnStutzenstein) [inline openslides_backend/action/motion/create.py:43]: We do need a state id, so in the else-part, you must return an error.

Comment (reiterl) [reply openslides_backend/action/motion/create.py:44]: Add an else branch which throws an error.
Add a test for this.

File: openslides_backend/action/motion/create.py [hunk 2]
```diff
@@ -1,7 +1,53 @@
...
 from ..action import register_action
-from ..base import DummyAction
+from ..default_schema import DefaultSchema
+from ..generics import CreateAction
 
 
 @register_action("motion.create")
-class MotionCreate(DummyAction):
-    pass
+class MotionCreate(CreateAction):
```
Comment (jsangmeister) [inline openslides_backend/action/motion/create.py:12]: Please subclass from `CreateActionWithAgendaItem` (or whatever I called it) and add tests for agenda item creation.

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Please add a test with all optional fields.

**Comment by jsangmeister** [comment]
Other thing still missing:

- Fields: text, lead_motion_id, statute_paragraph_id, reason
  - amendment_paragraph_$<paragraph_number> will probably need special handling (see #170), so you can leave it out for now
  - if the special handling of some values is not clear from the comments in https://github.com/OpenSlides-action-crew/openslides-backend-coordination/issues/24, ask @FinnStutzenstein or leave it out for now
- submitter_ids: Here, user ids are given. You have to execute an `motion_submitter.create` action for each user and save the motion_submitter ids in this field. You can leave it out for now if you want.
- origin_id: Leave it out for now. Will be important later when we start to support mutiple meetings

File: openslides_backend/action/motion/create.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/motion/create.py]: Added the CreateActionWithAgendaItemMix here and add a test

**Comment by reiterl** [comment]
Update motion.create: add AgendaItem. Rebase, update tests.

**Comment by jsangmeister** [comment]
@reiterl please implement the changes from my comments above. A test with all fields is still missing and the fields mentioned in the other comment as well. If you have any questions, feel free to ask, or note here what you left out and why.

**Comment by normanjaeckel** [comment]
@reiterl Ping.

**Comment by reiterl** [comment]
Add another commit with the special fields and some tests.
Missing **amendment_paragraph_$, submitter_ids**.

**Comment by jsangmeister** [comment]
Ok. Will do the rest

**Comment by jsangmeister** [comment]
Fixed handling as described in https://github.com/OpenSlides-action-crew/openslides-backend-coordination/issues/24 and added amendmend_paragraphs and motion_submitters. Sets up on #264. @normanjaeckel please check the changes in my commit

**Review by normanjaeckel — APPROVED** [review]
Looks good but I can not think about it now.

**Review by jsangmeister — APPROVED** [review]
Merging this for now. If we notice bugs, we can fix them in a later PR.

**Pull request merged**